12:02:04 [DBHandler]  MainThread:{4406568384} [__init__()]   INFO 
 DBHandler was started in ThreadSafeMode. 

12:02:04 [DBHandler]  MainThread:{4406568384} [connect()]   INFO 
 DB ('7614_3497_stats_bloggerCorpus_en_extern_plaintext') was connected. 

12:02:04 [CLI]  MainThread:{4406568384} [stats()]  ERROR 
 Given corp_fname ('7614_corp_bloggerCorpus_en_extern_plaintext.db') wasn't found and can not be opened. 

